node/protocols/data_verification/data_verification.go

```diff
@@ -1756,7 +1757,13 @@ func (d *Protocol) handleIncomingFileTransfer(s network.Stream) {
 	// send the data to verifier
 	// TODO: handle a way to prevent concurrent sent of data to verifier.
 	go func() {
-		err := d.SendKeyIVRandomizedFileSegmentsAndDataToVerifier(context.Background(), verifierID, fileMetadata.FilePath, contractHash, fileTransferRequest.FileHash)
+		bo := backoff.NewExponentialBackOff()
+		bo.MaxInterval = time.Second * 1
+		bo.MaxElapsedTime = time.Second * 10
```

Comment:
Hi, thanks for the PR.
I think this line could be problematic. Does MaxElapsedTime mean if the function is still sending data after 10 seconds this will be retired?

Reply:
no, it means that it is the max allowed time for retries but an attempt itself may last longer
> After MaxElapsedTime the ExponentialBackOff stops.

Reply:
`SendKeyIVRandomizedFileSegmentsAndDataToVerifier` sends 3 slices (key, iv and file segments) + some unencrypted file data which is calculated based on the total encrypted file segments percentage number. This data will be variable depending on the file size, could be few bytes or few hundred MBs